packages/api/tests/lambdas/test-bulk-operation.js

```diff
@@ -784,11 +804,18 @@ test.serial('bulk operation BULK_GRANULE_REINGEST reingests granule IDs returned
   t.true(esSearchStub.called);
   t.is(reingestStub.callCount, 2);
 
-  reingestStub.args.forEach((callArgs) => {
+  reingestStub.args.forEach(async (callArgs) => {
     const matchingGranule = granules.find((granule) =>
       granule.granuleId === callArgs[0].granule.granuleId);
 
-    t.deepEqual(matchingGranule, callArgs[0].granule);
+    const pgGranule = await getUniqueGranuleByGranuleId(knex, matchingGranule.granuleId);
```

Comment:
Same as above: I think this step is unnecessary

Reply:
It's still necessary since translatePostgresGranuletoApiGranule requires a PG record to translate